src/Feldspar/Core/Interpretation.hs

```diff
@@ -40,6 +40,33 @@
 
 -- | Defines different interpretations of Feldspar programs
 
+{-# LANGUAGE CPP #-}
+
+#if 0
```

Comment:
Should be handled by the build system.

Reply:
Temporary solution. Will be removed in the next commit.